### ISSUE TITLE
Invalid color does not meet accessibility contrast requirements

### ISSUE THREAD

**Issue by jestelle** [PolymerElements/paper-input#326]
In:
https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.html
goog-red-500 is used as the default 'invalid' color, used to style text indicating an input is invalid.

However, at the default font size, goog-red-500 does not have needed contrast on a white background.
http://webaim.org/resources/contrastchecker/?fcolor=DB4437&bcolor=ffffff

The Material Design team recommends replacing the default with:
Google Red a700 (#D50000) on light backgrounds and
#ff6e6e on dark backgrounds.

#FF6E6E is a value outside of the Material palette but its the closest to Red A700 that passes accessibility.

Developers and designers are welcome to override these defaults, but should take care in meeting accessibility requirements within their own applications.

**Comment by notwaldorf** [comment]
We just made a change where we were globally going to set the error color to be `--paper-deep-orange-a700`, which seems to be the spec. Is that in anyway better? (Just so we make the correct change)

**Comment by cherukumilli** [comment]
@notwaldorf 
Do you know when this change is going to be pushed to master?

I currently don't see  `--error-color` defined anywhere in the polymer project. I don't see the invalid error text or the underline in the `--error-color` when there is an error on the field.

i.e., `paper-input-container` default invalid color is not working as you may already know.

Please see the `Inputs can validate...` section of the `paper-input` demo at `https://elements.polymer-project.org/elements/paper-input?view=demo:demo/index.html` to recreate the problem.

**Comment by notwaldorf** [comment]
Sorry about that. `paper-input` was missing an import -- just tagged a new release, which should fix it.

**Comment by cherukumilli** [comment]
@notwaldorf 
The demo @ https://elements.polymer-project.org/elements/paper-input?view=demo:demo/index.html&active=paper-input-container is still not working. It is not changing the error text color to --paper-deep-orange-a700.

**Comment by notwaldorf** [comment]
The catalog has not been updated yet. If you try it locally, it should work
fine.

**Comment by notwaldorf** [comment]
Closing, as this issue is old and looks fixed
